pkg/common/telemetry/common/sanitize.go

```diff
@@ -0,0 +1,22 @@
+package common
```

Comment:
This would probably be better put in `pkg/common/telemetry/labelutils` or something similar. The `common` stutter isn't great and the name `telemetry/common` in the presence of other packages like `telemetry/agent` sends signals the package contains metrics that are common to both server and agent, which is not the case here.

If we wrap the *WithLabels functions then this point is moot - these could be defined as unexported functions on the MetricsImpl struct

Reply:
spoke offline for clarity, will fix in next push